tests/test_grainstats_minicircle.py

```diff
@@ -19,7 +19,7 @@ def test_grainstats_regression(regtest, minicircle_grainstats: GrainStats) -> No
     """Regression tests for grainstats."""
     statistics = minicircle_grainstats.calculate_stats()
     print(statistics)
```

Comment:
Pedantic but this line can be replaced, its likely a debugging leftover (probably mine!) as its the next line that saves the file using the `pytest-regtest` plugin.

Reply:
I've removed this and pushed to this branch.